src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -151,8 +151,7 @@ void PickFirst::HandOffPendingPicksLocked(LoadBalancingPolicy* new_policy) {
   while ((pick = pending_picks_) != nullptr) {
     pending_picks_ = pick->next;
     if (new_policy->PickLocked(pick)) {
-      // Synchronous return, schedule closure.
```

Comment:
Let's not lose this comment; it's still useful for the reader to know that the reason we're invoking the callback is that the pick completed synchronously.